sdk/containerregistry/Azure.Containers.ContainerRegistry/README.md

```diff
@@ -89,14 +95,88 @@ Uri endpoint = new Uri(Environment.GetEnvironmentVariable("REGISTRY_ENDPOINT"));
 // Create a new ContainerRegistryClient
 ContainerRegistryClient client = new ContainerRegistryClient(endpoint, new DefaultAzureCredential());
 
-// Perform an operation
+// Get the collection of repository names from the registry
 Pageable<string> repositories = client.GetRepositoryNames();
 foreach (string repository in repositories)
 {
     Console.WriteLine(repository);
 }
 ```
 
+### List tags with anonymous access
+
+```C# Snippet:ContainerRegistry_Tests_Samples_ListTagsAnonymous
+// Get the service endpoint from the environment
+Uri endpoint = new Uri(Environment.GetEnvironmentVariable("REGISTRY_ENDPOINT"));
+
+// Create a new ContainerRegistryClient for anonymous access
+ContainerRegistryClient client = new ContainerRegistryClient(endpoint);
+
+// Obtain a RegistryArtifact object to get access to image operations
+RegistryArtifact image = client.GetArtifact("library/hello-world", "latest");
+
+// List the set of tags on the hello_world image tagged as "latest"
+Pageable<ArtifactTagProperties> tags = image.GetTagPropertiesCollection();
+
+// Iterate through the image's tags, listing the tagged alias for the image
+Console.WriteLine($"{image.FullyQualifiedReference} has the following aliases:");
+foreach (ArtifactTagProperties tag in tags)
+{
+    Console.WriteLine($"    {image.RegistryEndpoint.Host}/{image.RepositoryName}:{tag}");
+}
+```
+
+### Set artifact properties
+
+```C# Snippet:ContainerRegistry_Tests_Samples_SetArtifactProperties
+// Get the service endpoint from the environment
+Uri endpoint = new Uri(Environment.GetEnvironmentVariable("REGISTRY_ENDPOINT"));
+
+// Create a new ContainerRegistryClient and RegistryArtifact to access image operations
+ContainerRegistryClient client = new ContainerRegistryClient(endpoint, new DefaultAzureCredential());
+RegistryArtifact image = client.GetArtifact("library/hello-world", "v1");
+
+// Set permissions on the v1 image's "latest" tag
+image.UpdateTagProperties("latest", new ArtifactTagProperties()
+{
+    CanWrite = false,
+    CanDelete = false
+});
+```
+
+### Delete images
```

Comment:
There is work in progress to parallelize this sample, but it won't be complete in time for Beta 3.  https://github.com/azure/azure-sdk-for-net/issues/21497